common/src/main/java/com/viaversion/viaversion/protocols/protocol1_16to1_15_2/packets/InventoryPackets.java

```diff
@@ -203,7 +203,7 @@ public static void oldToNewAttributes(Item item) {
             NumberTag leastTag = attribute.getNumberTag("UUIDLeast");
```

Comment:
I suppose if we want to remove data from the old protocol, you should use remove() for both UUIDLeast and UUIDMost?

Reply:
Makes sense to me. Added